llarp/config/config.cpp

```diff
@@ -1114,7 +1114,10 @@ namespace llarp
   void
   ApiConfig::defineConfigOptions(ConfigDefinition& conf, const ConfigGenParameters& params)
   {
-    constexpr Default DefaultRPCBindAddr{"tcp://127.0.0.1:1190"};
+    constexpr std::array DefaultRPCBind{
+        Default{"tcp://127.0.0.1:1190"},
+        Default{"ipc://rpc.sock"},
```

Comment:
I think we probably want to not include the latter one here as a default for windows.  I know there is, in theory, support for unix sockets in Windows 10 since sometime in 2018 or so, but I have no idea if it actually *works* via the rest of our software stack (zmq, etc.).

So I think let's just remove it from the defaults for windows, at least until someone has the time and energy to test it and figure out how to get windows to tell us whether it actually supports it yet.  (The only docs I can find on how to tell involve opening a windows command prompt as admin and running some "sc" comment, which is just... no.)
```suggestion
#ifndef _WIN32
        Default{"ipc://rpc.sock"},
#endif
```